tracker/event_tracker_store_test.go

```diff
@@ -35,14 +35,10 @@ func createSetupDB(subscriber eventSubscription, numBlockConfirmations uint64) s
 }
 
```

Comment:
Parallel execution is removed, because ethgo.Log MarshalJSON function is not thread-safe (it relies on global variable for `fastjson.ArenaPool`):

```go
WARNING: DATA RACE
Read at 0x00c000496600 by goroutine 9:
  github.com/valyala/fastjson.(*Arena).NewString()
      /home/stefan/go/pkg/mod/github.com/valyala/fastjson@v1.6.3/arena.go:61 +0x2c7
  github.com/umbracle/ethgo.(*Log).MarshalJSON()
      /home/stefan/go/pkg/mod/github.com/umbracle/ethgo@v0.1.4-0.20230810113823-c9c19bcd8a1e/structs_marshal.go:24 +0x546
  github.com/0xPolygon/polygon-edge/tracker.(*Entry).StoreLogs.func1()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store.go:254 +0x129
  go.etcd.io/bbolt.(*DB).Update()
      /home/stefan/go/pkg/mod/go.etcd.io/bbolt@v1.3.7/db.go:862 +0xda
  github.com/0xPolygon/polygon-edge/tracker.(*Entry).StoreLogs()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store.go:247 +0x118
  github.com/umbracle/ethgo/tracker/store.testMultipleStores()
      /home/stefan/go/pkg/mod/github.com/umbracle/ethgo@v0.1.4-0.20230810113823-c9c19bcd8a1e/tracker/store/testing.go:33 +0x219
  github.com/umbracle/ethgo/tracker/store.TestStore()
      /home/stefan/go/pkg/mod/github.com/umbracle/ethgo@v0.1.4-0.20230810113823-c9c19bcd8a1e/tracker/store/testing.go:15 +0x36
  github.com/0xPolygon/polygon-edge/tracker.TestBoltDBStore()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store_test.go:40 +0x49
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1576 +0x216
  testing.(*T).Run.func1()
      /usr/local/go/src/testing/testing.go:1629 +0x47

Previous write at 0x00c000496600 by goroutine 14:
  github.com/valyala/fastjson.(*Arena).Reset()
      /home/stefan/go/pkg/mod/github.com/valyala/fastjson@v1.6.3/arena.go:27 +0x3a
  github.com/umbracle/ethgo.(*Log).MarshalJSON.func1()
      /home/stefan/go/pkg/mod/github.com/umbracle/ethgo@v0.1.4-0.20230810113823-c9c19bcd8a1e/structs_marshal.go:16 +0x39
  runtime.deferreturn()
      /usr/local/go/src/runtime/panic.go:476 +0x32
  github.com/0xPolygon/polygon-edge/tracker.(*Entry).StoreLogs.func1()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store.go:254 +0x129
  go.etcd.io/bbolt.(*DB).Update()
      /home/stefan/go/pkg/mod/go.etcd.io/bbolt@v1.3.7/db.go:862 +0xda
  github.com/0xPolygon/polygon-edge/tracker.(*Entry).StoreLogs()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store.go:247 +0x118
  github.com/0xPolygon/polygon-edge/tracker.TestEventTrackerStore_SetLastBlockSubscriberNotified()
      /home/stefan/go/src/polygon-edge/tracker/event_tracker_store_test.go:158 +0x421
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1576 +0x216
  testing.(*T).Run.func1()
      /usr/local/go/src/testing/testing.go:1629 +0x47
```